xcode/Subconscious/Shared/Components/Common/ExpandAlignedLeadingViewModifier.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ExpandHorizontal.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 3/4/23.
+//
+
+import SwiftUI
+
+/// Modify a view so that it expands to fill the whole available
+/// horizontal space, with content aligned leading.
+struct ExpandAlignLeadingViewModifier: ViewModifier {
+    func body(content: Content) -> some View {
+        HStack(spacing: 0) {
+            content
+            Spacer()
+        }
+    }
+}
+
+extension View {
+    /// Modify a view so that it expands to fill the whole available
+    /// horizontal space, with content aligned leading.
+    func expandAlignedLeading() -> some View {
```

Comment:
Causes view to fill the whole row, with content aligned leading. I find myself doing this a lot and its tedious. Add a view modifier extension to accomplish.